### PR TITLE
added local maven finix library and test implementation finix library

### PR DESCRIPTION
**Pull request by tariq-g**

### CODE REVIEW

**Comment by tariq-g** [comment]
This will make it easier to test library changes on the fly

File: app/build.gradle [hunk 1]
```diff
@@ -90,7 +90,9 @@ dependencies {
 
 
     //Finix Payment Library
-    implementation files('libs/app-release.aar')
+//    implementation files('/Users/tariq.ebadi/AndroidStudioProjects/finix-android-payment-sheet/finix-android-payment-sheet/build/outputs/aar/finix-android-payment-sheet-release.aar')
```
Comment (cenobioh) [inline app/build.gradle:93]: you probably don't want to expose these local file names within the public app

**Review by cenobioh — CHANGES_REQUESTED** [review]
remove local file references in the code

File: app/build.gradle (hunk 1 above)
Comment (tariq-g) [reply app/build.gradle]: updated with comments removed

**Review by cenobioh — APPROVED** [review]

**Pull request merged**